### PR TITLE
[monorepo] task: add jacoco support

### PR DESCRIPTION
**Pull request by Wayonb**
## What is the current behavior?
There is no support for Java code coverage.

## What's the issue?
Java code coverage is not getting verified.

## How have you changed the behavior?
Jacoco option is added to verify the code coverage meets what is configured in the pom.xml

### CODE REVIEW

File: jenkins/shared-library/vars/codeCoverage.groovy [hunk 1]
```diff
@@ -11,17 +11,26 @@ void uploadCodeCoverage(String flag) {
...
 			runScript("coverage report --fail-under=${target}")
 		},
-		'nyc': { target ->
+		'nyc': { Integer target ->
+			logCodeCoverageMinimum(target)
 			runScript('npx nyc@latest report --lines')
 			runScript("npx nyc@latest check-coverage --lines ${target}")
+		},
+		'jacoco': { Integer target ->
+			logger.logInfo('Minimum code coverage is set pom.xml')
```
Comment (Jaguar0625) [inline jenkins/shared-library/vars/codeCoverage.groovy:31]: set *in* pom? should you read from pom and display result here?

Comment (Wayonb) [reply jenkins/shared-library/vars/codeCoverage.groovy:31]: yea, I was thinking that also.
Or to keep the code coverage consistent across all tools, take the minimumCodeCoverage and overwrite the value in the pom.  Any preference?

Comment (Jaguar0625) [reply jenkins/shared-library/vars/codeCoverage.groovy:31]: overwriting is fine too, but in that case would probably put 0 in pom with comment

Comment (Wayonb) [reply jenkins/shared-library/vars/codeCoverage.groovy:31]: Will read the value and display it here.  Thanks.

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**